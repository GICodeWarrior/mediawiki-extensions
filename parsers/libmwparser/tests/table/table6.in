<table> <td> table1 </table>
{|
| table2
|}
|
|}
<table> <td> table3 </table>
</table>
<table> <td> table4
{|
| table5 </table>
{|
| table6
<table> <td> table7
|}
</table>
|}

