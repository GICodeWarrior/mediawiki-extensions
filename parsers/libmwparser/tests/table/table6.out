BEGIN ARTICLE
    SPACE[ ]
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    WORD[table]
                    WORD[1]
                    SPACE[ ]
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    WORD[table]
                    WORD[2]
                    NEWLINE
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
    BEGIN PARAGRAPH
        SPECIAL[|]
        NEWLINE
        SPECIAL[|]
        SPECIAL[}]
    END PARAGRAPH
    SPACE[ ]
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    WORD[table]
                    WORD[3]
                    SPACE[ ]
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
    BEGIN PARAGRAPH
        SPECIAL[</]
        WORD[table]
        SPECIAL[>]
    END PARAGRAPH
    SPACE[ ]
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    WORD[table]
                    WORD[4]
                    NEWLINE
                    BEGIN TABLE
                        BEGIN TABLE BODY
                            BEGIN ROW
                                BEGIN CELL
                                    SPACE[ ]
                                    WORD[table]
                                    WORD[5]
                                    SPACE[ ]
                                    SPECIAL[</]
                                    WORD[table]
                                    SPECIAL[>]
                                    NEWLINE
                                    BEGIN TABLE
                                        BEGIN TABLE BODY
                                            BEGIN ROW
                                                BEGIN CELL
                                                    SPACE[ ]
                                                    WORD[table]
                                                    WORD[6]
                                                    NEWLINE
                                                    SPACE[ ]
                                                    BEGIN TABLE
                                                        BEGIN TABLE BODY
                                                            BEGIN ROW
                                                                BEGIN CELL
                                                                    SPACE[ ]
                                                                    WORD[table]
                                                                    WORD[7]
                                                                    NEWLINE
                                                                    SPECIAL[|]
                                                                    SPECIAL[}]
                                                                    NEWLINE
                                                                END CELL
                                                            END ROW
                                                        END TABLE BODY
                                                    END TABLE
                                                END CELL
                                            END ROW
                                        END TABLE BODY
                                    END TABLE
                                END CELL
                            END ROW
                        END TABLE BODY
                    END TABLE
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
END ARTICLE
